validator-api/validator_api/services/scoring_service.py

```diff
@@ -266,19 +271,58 @@ async def get_detailed_video_description_embedding_score(self, video_id, task_ov
             return detailed_video_description, None, None
         return detailed_video_description, embedding, await self.get_description_uniqueness_score(embedding)
 
+    async def get_boosted_multiplier(self, focusing_task: str, focusing_description: str) -> float:
+        """
+        Get boosted tasks from the database "boosted_tasks" table
+        ask Gemini if the task matches any of the boosted tasks
+        return the multiplier for the task if it is boosted, otherwise 1.0
+        """
+        with get_db_context() as db:
+            boosted_tasks_records = db.query(BoostedTask).all()
+            # Convert to simple tuples containing just the data we need
+            boosted_tasks = [(task.title, task.description, task.multiplier) for task in boosted_tasks_records]
+        # print(f"Boosted tasks: {boosted_tasks}")
+        system_prompt = focus_scoring_prompts.BOOST_SCORING_SYSTEM_PROMPT.format(
+            boosted_tasks="\n".join([f"{idx}. {title}: {description}" for idx, (title, description, _) in enumerate(boosted_tasks)])
+        )
+        user_prompt = focus_scoring_prompts.BOOST_SCORING_USER_PROMPT.format(
+            focusing_task=focusing_task,
+            focusing_description=focusing_description,
+        )
+        boosted_task_index = await self.make_gemini_request_with_retries(
+            system_prompt=system_prompt,
+            user_prompt=user_prompt,
+            video_id=None,
+            OutputClassSchema=BoostedTaskIndex,
+        )
+        # print(f"Boosted task index: {boosted_task_index.index}")
+        if boosted_task_index.index == -1 or boosted_task_index.index >= len(boosted_tasks):
+            return 1.0
+            
+        multiplier = boosted_tasks[boosted_task_index.index][2]  # Get multiplier from tuple
```

Comment:
pydantic objects are more readable and less error prone than using tuple indexing